lib/OpenQA/Worker/Job.pm

```diff
@@ -124,30 +124,17 @@ sub _set_status {
     $self->emit(status_changed => $event_data);
 }
 
-sub is_stopped_or_stopping {
-    my ($self) = @_;
+sub is_stopped_or_stopping { shift->status =~ /stopped|stopping/ }
 
-    my $status = $self->status;
-    return $status eq 'stopped' || $status eq 'stopping';
```

Comment:
Don't change that for the sake of saving a few lines. The real improvement would be using constants in my opinion.

Reply:
I left out this change for now